types/sinon-sandbox/tsconfig.json

```
@@ -0,0 +1,23 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6"
```

```suggestion
{
    "compilerOptions": {
        "module": "commonjs",
        "lib": [
            "es6",
            "DOM"
```
this will fix that dependency resolution problem during CI tests (that is same dependency as in `sinonjs` types.